docker-compose.yml

```diff
@@ -0,0 +1,7 @@
+web:
+    build: .
+    command: bundle exec rails s -p 3000 -b '0.0.0.0'
+    volumes:
+        - .:/workspace
+    ports:
+        - "3000"
```

Comment:
There is necessary `- "3000:3000"`.

It was tested in Ubuntu 16.04LTS, kernel 4.4, Docker 1.13.1.